src/reducers/currentFilm.js

```diff
@@ -0,0 +1,6 @@
+const current = (state, action) => ({
```

Comment:
reducers have to react on a specific action and return new state only if they can handle action. Your reducers everytime return new state. Redux calls all reducers when action is occured.